Cannot find module 'debug'

I just updated to flexi@1.1.4 and am getting this error with `ember serve`.

> Cannot find module 'debug'
> Error: Cannot find module 'debug'
>     at Function.Module._resolveFilename (module.js:336:15)
>     at Function.Module._load (module.js:278:25)
>     at Module.require (module.js:365:17)
>     at require (module.js:384:17)
>     at Object.<anonymous> (/Users/gcheung/dev/proto/fitbit-site-ui/node_modules/flexi/htmlbars-plugins/attribute-conversion.js:26:13)
>     at Module._compile (module.js:460:26)
>     at Object.Module._extensions..js (module.js:478:10)
>     at Module.load (module.js:355:32)
>     at Function.Module._load (module.js:310:12)
>     at Module.require (module.js:365:17)
>     at require (module.js:384:17)

I believe it's because debug is in `devDependencies` instead of `dependencies`.

I just created a new ember app 3 times in a row just to make sure it wasn't something else.
Then I installed flexi thru `ember install flexi` and seems like a bug like the one above I am encountering everytime:
```
Installed packages for tooling via npm.
Cannot find module 'debug'
Error: Cannot find module 'debug'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/Users/greyhwndz/@dev/flexi-app3/node_modules/flexi/htmlbars-plugins/attribute-conversion.js:26:13)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

--- a/package.json
+++ b/package.json
@@ -21,7 +21,6 @@
   "devDependencies": {
     "broccoli-asset-rev": "^2.2.0",
     "broccoli-stew": "^1.2.0",
-    "debug": "^2.2.0",
     "ember-cli": "2.4.2",
     "ember-cli-app-version": "^1.0.0",
     "ember-cli-changelog": "0.3.4",
@@ -64,6 +63,7 @@
     "broccoli-merge-trees": "^1.1.1",
     "broccoli-plugin": "^1.2.1",
     "chalk": "^1.1.1",
+    "debug": "^2.2.0",
     "ember-cli-babel": "^5.1.5",
     "ember-cli-htmlbars": "^1.0.1",
     "ember-cli-preprocess-registry": "^3.0.0",

